haas/cli.py

```diff
@@ -59,6 +59,8 @@ def request(method, url, data=None):
             The URL to act on
         data : str, optional
             The body of the request
+        params : str, optional
+            The GET parameter
```

Comment:
This should be a dictionary; see <http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls>

Also, s/GET parameter/query parameters/ as above.